### PR TITLE
Milestone 3 - Minimal Quiz

### PR DESCRIPTION
**Pull request by abdulrauf11**
The PR provides a build that demonstrates a fully functional Quiz Screen.

## Tasks
- The global settings of the application can be updated through the `/src/config.json` file.
- As discussed, the skip button has been replaced by the space key bar instead of a mouse click.
- All the major parts of the quiz have been implemented. These include player inputs, score updates, question countdown, quiz progress, 3 types of questions, question set cycle, dynamic content/configuration.

## To Do
- Styling
- Captions for all the audio/video files
- Sound effects used throughout the application
- Update with final assets
- Code cleanup and optimization
- Some edge cases for the quiz

### Edge cases
1. If all players enter answers before audio narration completes, the results are shown as soon as the audio narration completes.
2. When appropriate, the media asset updates to display the "reveal" answer version.
3. Inactivity timeout - if no answers are entered for two quiz questions in a row, the application reloads/resets to the Attract screen.
4. Software must be able to run in an exhibit for ~12 hours without any performance issues. You may reload the page programmatically after a period of inactivity to help avoid memory issues.

Let me know if we have missed an edge case.

## Test Link
https://tsck-bionic-eyes-quiz-show.netlify.app/

## Preview
Screenshot 2021-08-19 at 5 06 24 PM

### CODE REVIEW

File: src/config.json [hunk 1]
```diff
@@ -8,5 +8,16 @@
     "wrong": "/path/to/wrong/answer/sound",
     "beforeTimer": "/path/to/players/answer/before/timer/sound",
     "afterTimer": "/path/to/players/answer/after/timer/sound"
+  },
+
+  "controls": {
```
Comment (tnordberg) [inline src/config.json:13]: Smart. Thanks for making these configurable.

**Review by tnordberg — APPROVED** [review]
@abdulrauf11 all is approved now. Thanks again for the detailed PR description.
